### PR TITLE
Tabulated eos evolution

### PR DESCRIPTION
**Pull request by SamuelCupp**
Debugged and tested version of tabulated EOS

### CODE REVIEW

File: GRHayL/Flux_Source/ghl_calculate_characteristic_speed_dirn0.c
Comment (SamuelCupp) [inline GRHayL/Flux_Source/ghl_calculate_characteristic_speed_dirn0.c:1]: There's no reason to remove the const from this function.

File: GRHayL/Flux_Source/ghl_calculate_source_terms_dirn0.c
Comment (SamuelCupp) [inline GRHayL/Flux_Source/ghl_calculate_source_terms_dirn0.c:1]: There's no reason to remove the const from this function.

File: GRHayL/Flux_Source/ghl_calculate_tau_tilde_source_term_extrinsic_curv.c
Comment (SamuelCupp) [inline GRHayL/Flux_Source/ghl_calculate_tau_tilde_source_term_extrinsic_curv.c:1]: There's no reason to remove the const from this function.

File: GRHayL/Flux_Source/tabulated/grhayl_calculate_characteristic_speed_dirn0.c
Comment (SamuelCupp) [inline GRHayL/Flux_Source/tabulated/grhayl_calculate_characteristic_speed_dirn0.c:1]: We don't need tabulated versions of these functions, correct?

File: GRHayL/Flux_Source/tabulated/grhayl_calculate_source_terms_dirn0.c
Comment (SamuelCupp) [inline GRHayL/Flux_Source/tabulated/grhayl_calculate_source_terms_dirn0.c:1]: We don't need tabulated versions of these functions, correct?

File: GRHayL/Flux_Source/tabulated/grhayl_calculate_tau_tilde_source_term_extrinsic_curv.c
Comment (SamuelCupp) [inline GRHayL/Flux_Source/tabulated/grhayl_calculate_tau_tilde_source_term_extrinsic_curv.c:1]: We don't need tabulated versions of this function, correct?

Comment (leowerneck) [reply GRHayL/Flux_Source/tabulated/grhayl_calculate_tau_tilde_source_term_extrinsic_curv.c:1]: Correct; latest commit (fb8c62b..2f9012d) removes these.

**Pull request merged**